src/util/outputNameTemplate.js

```diff
@@ -65,15 +65,23 @@ export function getOutSegError({ fileNames, filePath, outputDir, safeOutputFileN
 // This is used as a fallback and so it has to always generate unique file names
 // eslint-disable-next-line no-template-curly-in-string
 export const defaultOutSegTemplate = '${FILENAME}-${CUT_FROM}-${CUT_TO}${SEG_SUFFIX}${EXT}';
-
+let currentTimestamp = Date.now();
 function interpolateSegmentFileName({ template, inputFileNameWithoutExt, segSuffix, ext, segNum, segLabel, cutFrom, cutTo, tags }) {
   const compiled = lodashTemplate(template);
+
+  if (segSuffix) {
+    if (segNum > 1) {
+      currentTimestamp += 1;
+    } 
+  }
```

Comment:
could you explain this? shouldn't it instead be:
```js
currentTimestamp += segNum
```

Reply:
I reviewed your changes I think your idea to use `currentTimestamp + i` ([this line](https://github.com/solderq35/lossless-cut/blob/1e5dbb6f9e3078dcb249c90124433868ce366e85/src/util/outputNameTemplate.js#L117)) to be a good approach as well.